prediction_market_agent/agents/gptr_agent/deploy.py

```
@@ -8,18 +8,25 @@
 from prediction_market_agent_tooling.gtypes import USD
+from prediction_market_agent_tooling.gtypes import USD
 from prediction_market_agent_tooling.markets.agent_market import AgentMarket, SortBy
 from prediction_market_agent_tooling.markets.data_models import ProbabilisticAnswer
+from prediction_market_agent_tooling.markets.omen.omen import OmenAgentMarket
 from prediction_market_agent_tooling.tools.langfuse_ import observe
 from pydantic_ai import Agent
 from pydantic_ai.models.openai import OpenAIModel
 from pydantic_ai.settings import ModelSettings
 
+from prediction_market_agent.agents.utils import get_maximum_possible_bet_amount
 from prediction_market_agent.agents.utils import get_maximum_possible_bet_amount
```

Ruff (0.12.2)

Redefinition of unused `get_maximum_possible_bet_amount` from line 29

Remove definition: `get_maximum_possible_bet_amount`

(F811)